ocaml/xapi/xapi_xenops.ml

```diff
@@ -271,16 +271,17 @@ let builder_of_vm ~__context ~vm timeoffset pci_passthrough =
 			and command = 3 (* IO and memory access *)
 			and status = 0
 			and revision = 1
-			and classcode = 0x000100 (* Storage device class, SCSI subclass *)
+			and classcode = 0x088000 (* Other system peripheral. See
```

Comment:
This is wrong... this should be `0x008000`. I would rather remove the preceding zeros and show the PCI class IDs for the 16bit integers that they are... i.e. `0x8000`. The MSbyte is the base class (0x80) and the LSbyte is the subclass (0x00).